src/main/io/osd.c

```diff
@@ -3378,7 +3384,7 @@ textAttributes_t osdGetSystemMessage(char *buff, size_t buff_size, bool isCenter
                 if (FLIGHT_MODE(NAV_RTH_MODE) || FLIGHT_MODE(NAV_WP_MODE) || navigationIsExecutingAnEmergencyLanding()) {
                     if (NAV_Status.state == MW_NAV_STATE_WP_ENROUTE) {
                         // Countdown display for remaining Waypoints
-                        tfp_sprintf(messageBuf, "TO WP %u/%u", posControl.activeWaypointIndex + 1, posControl.waypointCount);
+                        tfp_sprintf(messageBuf, "TO WP %u/%u (%3u M)", getGeoWaypointNumber(posControl.activeWaypointIndex), posControl.geoWaypointCount, (int)(posControl.wpDistance / 100));
```

Comment:
Would it fit to add the distance unit? For WP distance above 1km, it could show the distance in xx.x km format. Our American friends would probably appreciate empirical units.

Reply:
Ah good point. I'll need to check that.